content/blogs/Porting-Syzkaller-to-ReactOS-GSoC-2020.md

```diff
@@ -0,0 +1,32 @@
+---
+title:       "Porting Syzkaller to ReactOS - GSoC 2020"
+author:      "Freakston"
+date:        2020-05-27
+tags:        [ gsoc ]
+---
+
+Hey , I am Suraj K Suresh (@Freakston) , one of the GSoC students of ReactOS. I'm very excited to spend my summer contributing to ReactOS.  
```

Comment:
```suggestion
Hey, I am Suraj K Suresh (@Freakston), one of the GSoC students of ReactOS.
I'm very excited to spend my summer contributing to ReactOS.  
```

Reply:
By adding each line on a new line in the file, it is easier to read in the diffs and merge changes.

Reply:
Resolved and I will keep that in mind from the next blogposts.